lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -219,22 +219,21 @@ def log.trace?
         ca_cert = File.join(@secret_dir, K8_POD_CA_CERT)
         pod_token = File.join(@secret_dir, K8_POD_TOKEN)
 
-        if !@ca_file.present? and File.exist?(ca_cert)
+        if !present?(@ca_file) and File.exist?(ca_cert)
```

Comment:
These cause test failures.  Is this dependent upon your other PR to bump to 2.5?

Reply:
fixed